.github/workflows/docker-build.yml

```diff
@@ -425,22 +433,31 @@ jobs:
         shell: bash
         run: |
           PSK= RESOTOCORE_ANALYTICS_OPT_OUT=true docker-compose up -d
-          cd resoto.com/versioned_docs/version-${{steps.release.outputs.docsVersion}}/reference/data-models
+          cd resoto.com/versioned_docs/version-${{ steps.release.outputs.docsVersion }}/reference/data-models
           find . -mindepth 2 -type d -name img -prune -exec rm -rf {} \;
           python3 ${{ github.workspace }}/resoto.com/tools/export_model_images.py
 
       - name: Update released version & resotocore API YAML
         if: steps.release.outputs.prerelease == 'false'
         shell: bash
         run: |
-          cp resotocore/resotocore/static/api-doc.yaml resoto.com/openapi/resotocore-${{steps.release.outputs.docsVersion}}.yml
-          echo $(jq '.["${{steps.release.outputs.docsVersion}}"].version="${{ steps.release.outputs.tag }}" | .["${{steps.release.outputs.docsVersion}}"].link="${{ steps.release_notes.outputs.link }}"' resoto.com/latestRelease.json) > resoto.com/latestRelease.json
+          cp resotocore/resotocore/static/api-doc.yaml resoto.com/openapi/resotocore-${{ steps.release.outputs.docsVersion }}.yml
+          echo $(jq '.["${{ steps.release.outputs.docsVersion }}"].version="${{ steps.release.outputs.tag }}" | .["${{ steps.release.outputs.docsVersion }}"].link="${{ steps.release_notes.outputs.link }}"' resoto.com/latestRelease.json) > resoto.com/latestRelease.json
+
+      - name: Update AWS policy JSON
+        if: steps.release.outputs.prerelease == 'false'
+        shell: bash
+        working-directory: ./resoto.com/aws/${{ steps.release.outputs.docsVersion }}
+        run: |
```

Comment:
There is a race condition here between the AWS plugin workflow uploading and this workflow downloading the JSON files. Right now there's a good chance it will work, because the Docker build takes longer than the PIP build. But there's not guarantee for this code working.

Reply:
Yeah, I was thinking about how to best handle this.

We had discussed splitting the release/docs logic into a separate workflow previously, so I think this is a good time to make that change. See my latest commit :)